decidim-core/lib/tasks/decidim_metrics_tasks.rake

```diff
@@ -43,15 +43,16 @@ namespace :decidim do
         raise ArgumentError if day.blank?
 
         (Date.parse(day)..Time.zone.today).each do |d|
+          current_day = d.to_s
```

Comment:
`date.to_s(:db)` would it be safer?

Reply:
why? it is converted to string to be passed as aregument to the job and then parsed again. It never goes to DB as this string. I think it won't be safer...

Reply:
ok